### PR TITLE
Minor fix to service collection extensions

### PR DESCRIPTION
**Pull request by kichalla**
@Eilon

### CODE REVIEW

**Comment by Eilon** [comment]
:shipit:

File: .gitignore [hunk 1]
```diff
@@ -28,3 +28,4 @@ nuget.exe
 project.lock.json
 .build/
 .testPublish/
+launchSettings.json
```
Comment (Tratcher) [inline .gitignore]: We're really inconsistent about this. Some of the samples require customized launch settings (e.g. Security). I'd rather check them in everywhere.

**Comment by kichalla** [comment]
:up: :date:  Returning void now

**Comment by Eilon** [comment]
:shipit:

**Comment by kichalla** [comment]
46f6fa85d5c7366ee3a383cbe92c10146127f916

**Pull request closed without merging**